src/Carnac/App.xaml.cs

```diff
@@ -44,6 +56,30 @@ protected override void OnStartup(StartupEventArgs e)
             carnac = new KeysController(keyShowViewModel.Messages, messageProvider, new ConcurrencyService(), settingsProvider);
             carnac.Start();
 
+            Observable
+                .Timer(TimeSpan.FromMinutes(5))
+                .Subscribe(async x =>
+                {
+                    try
+                    {
+#if DEBUG
```

Comment:
Not much, I think it was just more for my benefit while testing. I'll just enable it for release builds